src/main/java/com/amihaiemil/docker/Container.java

```diff
@@ -57,4 +57,10 @@
      * @return String id.
      */
     String containerId();
+
+    /**
+     * Stops the container.
+     * @throws IOException If something goes wrong.
```

Comment:
@llorllale I think we should start specifying:

``
@throws UnexpectedResponseException If the status response is not <<expected>>
``
as well, not just ``IOException``.
Don't you think?

Reply:
@amihaiemil done